engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -341,4 +341,32 @@ - (int64_t)lastAppModificationTime {
   return [fileDate timeIntervalSince1970];
 }
 
+- (UISceneConfiguration*)application:(UIApplication*)application
+    configurationForConnectingSceneSession:(UISceneSession*)connectingSceneSession
+                                   options:(UISceneConnectionOptions*)options
+    API_AVAILABLE(ios(13.0)) {
+  NSDictionary* sceneManifest =
+      [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIApplicationSceneManifest"];
+  NSDictionary* sceneConfigs = sceneManifest[@"UISceneConfigurations"];
+  NSArray* roleConfigs = sceneConfigs[connectingSceneSession.role];
+
+  UISceneConfiguration* config;
+  if (roleConfigs.count > 0) {
+    NSString* configurationName = roleConfigs[0][@"UISceneConfigurationName"];
+    config = [UISceneConfiguration configurationWithName:configurationName
```

Comment:
Does this work? I thought we also need to parse the `sceneClass/delegateClass` etc from the dictionary and set it to the config.

Reply:
It turns out that we don't have to process the plist by ourselves, since UIKit already processed it before calling this function. See https://github.com/flutter/flutter/pull/167588/files#r2055190090

Reply:
Right, we are checking the plist if it exists, and if it exists this function grabs it.